syft/internal/cache/dir.go

```diff
@@ -0,0 +1,71 @@
+package cache
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/adrg/xdg"
+	"github.com/mitchellh/go-homedir"
+
+	"github.com/anchore/syft/internal/log"
+)
+
+// SetRootDir sets the root directory to use for downloaded content caching for things like module license information
+func SetRootDir(dir string) {
+	rootDir = dir
```

Comment:
should we always nest `syft` to whatever is passed?

Reply:
I don't believe so, if I tell syft to cache in directory like `~/syft-cache` it would be unnecessary to put a subdirectory of `syft`. I think if people are specifying this, they're managing a directory for syft caches specifically, so the nesting is unnecessary. And, of course, if they _want_ a subdirectory, they can add it themselves and use ~/syft-cache/syft`, etc..